### DISCUSSION TITLE
About Running alm

### DISCUSSION THREAD

**Discussion by alleskonner28** [ttadano/alamode#81] [Q&A] [answered]
Dear Dr. Tadano,

I am a beginner using ALAMODE. I would like to use it to extract the harmonic pattern using `alm` 
I am following the ["Install using conda"](https://alamode.readthedocs.io/en/latest/install.html#:~:text=the%20XYZ%20format.-,3.2.%20Install%20using%20conda%20(recommended%20for%20non%2Dexperts),-This%20option%20is) procedure to install the alamode. 
It produced succesfully all executable files, including `alm`.

However, as I tried to calculate it, the calculation is stopped as shown by output error below 

```
slurmstepd: error: execve(): /home/alamode/_build/alm: Permission denied
slurmstepd: error: execve(): /home/alamode/_build/alm: Permission denied
```
I don't know what the reason for this error is.

Could you give me a clue for solving this problem?

Thanks 

Yedija

**Comment by ttadano** [comment, marked as answer]
Hi.

I think this is not an error of `alm`. Maybe, you can find some relevant information by googling `slurmstepd: error: execve()`.

**Reply by alleskonner28** [reply]
Dear Dr. Tadano

The problem has been solved. 
Thank you very much for the suggestion
